docs/participate/01_staking/01_become_a_collator/04_session_keys.md

```diff
@@ -63,7 +63,7 @@ The `result` key is the HEX-encoded public part of the newly created session key
 <TabItem value="apps">
 
 In order to use the PolkadotJS Apps UI, the node WebSocket endpoint must be reachable.
-This can be done either by publicly exposing it with the `--ws-external` flag, which is discouraged, or by setting up an SSH tunnel for the WebSocket endpoint with `ssh -L 127.0.0.1:9933:127.0.0.1:9933 <user>@<server>`.
+This can be done either by publicly exposing it with the `--rpc-external` flag, which is discouraged, or by setting up an SSH tunnel for the WebSocket endpoint with `ssh -L 127.0.0.1:9933:127.0.0.1:9933 <user>@<server>`.
```

Comment:
Is 9933 or 9944 the port we should advertise?

Reply:
Only [9944](https://github.com/paritytech/substrate/pull/13384/files#diff-79991d75ead2aa9757d97be3307d93100616d08c554e4c97e6567e6c8c625f1dL63)